hledger/Hledger/Cli/CompoundBalanceCommand.hs

```diff
@@ -129,11 +131,16 @@ compoundBalanceCommand CompoundBalanceCommandSpec{..} opts@CliOpts{reportspec_=r
 
         -- when user overrides, add an indication to the report title
         -- Do we need to deal with overridden BalanceCalculation?
-        mtitleclarification = flip fmap mbalanceAccumulationOverride $ \case
-            PerPeriod | changingValuation -> "(Period-End Value Changes)"
-            PerPeriod                     -> "(Balance Changes)"
-            Cumulative                    -> "(Cumulative Ending Balances)"
-            Historical                    -> "(Historical Ending Balances)"
+        mtitleclarification = case (balancecalc_, balanceaccumulation, mbalanceAccumulationOverride) of
+            (CalcValueChange, PerPeriod,  _              ) -> Just "(Period-End Value Changes)"
+            (CalcValueChange, Cumulative, _              ) -> Just "(Period-End Value Changes)"
```

Comment:
Should the above titles not be different from each other ?

Reply:
Yes, they should.